cogs/SquadQueue.py

```diff
@@ -793,11 +952,9 @@ async def add_teams_to_rooms(self, mogi: Mogi, open_time: int, started_automatic
 
         extra_members = []
 
-        extra_members_ids = self.bot.config["members_for_channels"].get(str(mogi.mogi_channel.guild.id), [])
-        for m in extra_members_ids:
+        for m in self.bot.config["members_for_channels"]:
             extra_members.append(mogi.mogi_channel.guild.get_member(m))
-        extra_roles_ids = self.bot.config["roles_for_channels"].get(str(mogi.mogi_channel.guild.id), [])
-        for r in extra_roles_ids:
+        for r in self.bot.config["roles_for_channels"]:
```

Comment:
The main reason I used `.get` was in case these were missing from the configs. Although, if they were, the person running the bot probably has bigger issues. I think your change is best as we should both have valid configs.